### ISSUE TITLE
tl;dr?

### ISSUE THREAD

**Issue by jmuehlenbein** [MicrosoftDocs/microsoft-365-community#507]
Great content! I'd like to pass it along to some end users but the weird "tl;dr" as heading of the first section is a bit offputting. Not sure what it's supposed to say but I'm hoping it can be corrected? thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e7c49951-8871-c489-e4f3-81d845ebbce3
* Version Independent ID: c84959ec-3662-6963-00dc-5b3a0d9873c0
* Content: [Living Large with Large Lists and Large Libraries](https://docs.microsoft.com/en-us/microsoft-365/community/large-lists-large-libraries-in-sharepoint)
* Content Source: [Community/large-lists-large-libraries-in-SharePoint.md](https://github.com/MicrosoftDocs/microsoft-365-community/blob/master/Community/large-lists-large-libraries-in-SharePoint.md)
* Service: **sharepoint-online**
* GitHub Login: @PatD
* Microsoft Alias: **daisyfeller**

**Comment by sympmarc** [comment]
tl;dr means there's a lot of text and this is the summary. See:
https://www.urbandictionary.com/define.php?term=tl%3Bdr

If you think it is significantly off-putting, I suppose we could change it so say "Summary".

**Comment by jmuehlenbein** [comment]
Hey Marc, I should have known this was your work. really, great stuff! =)  I am obviously not up on the lingo of the kids these days so it's probably just fine but maybe ask a few other people if they know what it means? Sorry for assuming it was a mistake rather than just looking it up!

**Comment by sympmarc** [comment]
That article isn't my work, but I'm one of the admins for this repo.

Hey, since you had to ask, it's probably worth changing. We just did a push to live today, so it'll probably take a little while, but I'll change it.

We also use `tl;dr` at the top of the Maturity Model for Microsoft 365 articles, because they are even longer.

**Comment by jmuehlenbein** [comment]
Well, putting my communicator hat on, I'd say it doesn't set the best tone....has a negative connotation assuming the user knows what you mean, and is possibly confusing if (like me) they don't. Anyway, appreciate the opportunity to weigh in. =)
